src/components/Blog/BlogArticleCard/BlogArticleCard.tsx

```diff
@@ -66,7 +66,7 @@ export const BlogArticleCard = ({
       >
         {article.attributes?.Image.data ? (
           <BlogArticleCardImage
-            src={`${baseUrl}${article.attributes?.Image.data?.attributes?.formats.small.url}`}
+            src={`${baseUrl}${('formats' in article.attributes?.Image.data?.attributes && article.attributes?.Image.data?.attributes?.formats?.small.url) || article.attributes?.Image.data?.attributes?.url}`}
```

Comment:
if you make sure that formats exists with that statement, then the `?` right after it is not needed anymore